src/sparseml/utils/pytorch/module.py

```diff
@@ -96,6 +97,17 @@ def match_targets(name: str, targets: Union[str, List[str]]) -> Tuple[bool, int]
     return False, -1
 
 
+def match_class(layer: Module, targets: Union[str, List[str]]) -> Tuple[bool, int]:
+    if isinstance(targets, str):
+        targets = [targets]
+
+    for index, target in enumerate(targets):
+        if layer.__class__.__name__ == target:
+            return True, index
```

Comment:
why do we need a bool and index if returning -1 in the non-match case?

Reply:
I was just mimicking the style of the match_targets function above, it makes the if statement a bit cleaner on line 137 but I'm not opposed to using just the index